index.js

```diff
@@ -2,6 +2,13 @@
 require("dotenv").config();
 const axios = require("axios");
 
+// Usage from the command line:
+// npm install
+// node index sourceChannel targetChannel
+// e.g. node index general dev-chatter
+
+// Note: Do not include the hashtag in the channel name. Though I did account for stripping the hashtag away, process.argv returns undefined if the hashtag is present and I'm unsure of why.
```

Comment:
You also need to indicate how to get a hold of the Slack access token e.g. ping a collaborator to be added as a project collaborator, and then the page to go to after than (or a Slack link telling you how) to copy the access token generated.

Reply:
Is it appropriate to squeeze that sort of instruction into code comments rather than in onboarding documentation somewhere?

Reply:
@bhurstGH if I were reading this for the first time, I'd want that mentioned, if I'm trying to run this. Functionally, that's needed to make this run. So, that part of the process is required. If the link is to just an onboarding document, then that's fine too. However, considering we don't have that document yet (?) we could just tell them the two points above.